extensions-marketplace/develop/integrate/byteplus.mdx

```diff
@@ -0,0 +1,322 @@
+---
+title: "BytePlus"
+sidebar_position: 12
+type: docs
+description: >
+    BytePlus Effects extension encapsulates the core APIs of the BytePlus Effects SDK.
+---
+export const toc = [{}];
+
+This guide is provided by Voicemod. Agora is planning a documentation upgrade program for all extensions on the marketplace. Please stay tuned.
+
+<PlatformWrapper notAllowed="web">
+<PlatformWrapper notAllowed="macos">
+<PlatformWrapper notAllowed="windows">
+
+
+
+
+## Understand the tech
+
+The BytePlus Effects extension encapsulates the core APIs of the BytePlus Effects SDK. By calling the [setExtensionProperty](https://docs.agora.io/en/video-call-4.x/API Reference/java_ng/API/class_irtcengine.html#api_setextensionproperty) or [setExtensionPropertyWithVendor](https://docs.agora.io/en/video-call-4.x/API Reference/ios_ng/API/class_irtcengine.html#api_setextensionproperty) method of the [Agora Video SDK v4.0.0](https://docs.agora.io/en/video-call-4.x/product_video_ng?platform=Android) and passing in the corresponding `key` and `value`, you can quickly integrate BytePlus's capabilities.
```

Comment:
You need to update all of these links to use variables bla bla. 

Looking down, at a guess, most of these links are broken. If you could check throughout please.